openerp/exceptions.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, name, value=None):
             _logger.warn('except_orm is deprecated. Please use specific exceptions like UserError or AccessError. Caller: %s:%s', *caller)
         self.name = name
         self.value = value
+        value = value if value else ''
         self.args = (name, value)
```

Comment:
value = value if value else ''instead of this  pass 
self.args = (name, value or '')